index.html

```diff
@@ -50,5 +50,196 @@ <h1>Hey, I'm Aidan</h1>
+                    </div>
+                    <div class="language">
+                        <ul class="lang">
+                            <li class="lang-list"><button>Ruby on rails</button></li>
+                            <li class="lang-list"><button>css</button></li>
+                            <li class="lang-list"><button>JavaScript</button></li>
+                            <li class="lang-list"><button>html</button></li>
+                        </ul>
+                    </div>
+                    <div class="see">
+                        <button>See Project</button>
+                    </div>
+                </div>
+             </div>
+             <div class="work">
+                 <div class="multi-post">
+                     <div class="bottom">
+                          <p>Multi-Post Stories</p>
+                          <p>Gain+Glory</p>
+                     </div>
+                     <div class="language">
+                         <ul class="lang">
+                             <li class="lang-list"><button>Ruby on rails</button></li>
+                             <li class="lang-list"><button>css</button></li>
+                             <li class="lang-list"><button>JavaScript</button></li>
+                             <li class="lang-list"><button>html</button></li>
+                         </ul>
+                     </div>
+                     <div class="see">
+                         <button>See Project</button>
+                     </div>
+                 </div>
+              </div>
+              <div class="work">
+                 <div class="multi-post">
+                     <div class="bottom">
+                          <p>Multi-Post Stories</p>
+                          <p>Gain+Glory</p>
+                     </div>
+                     <div class="language">
+                         <ul class="lang">
+                             <li class="lang-list"><button>Ruby on rails</button></li>
+                             <li class="lang-list"><button>css</button></li>
+                             <li class="lang-list"><button>JavaScript</button></li>
+                             <li class="lang-list"><button>html</button></li>
+                         </ul>
+                     </div>
+                     <div class="see">
+                         <button>See Project</button>
+                     </div>
+                 </div>
+              </div>
+        </div>
+    </section>
+    <section>
+        <div class="aboutme">
+            <h1 class="recent">
+                About me
+            </h1>
+            <p class="aboutme-text">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+            <div class="resume">
+                <button>Get My Resume</button>
+            </div>
```

Comment:
- [ ] Your about me section background does not seem to be showing up. Additionally, the `margin-top` is not the same as in the Figma design. Please refer to the images below and resolve these issues
> Figma
![image](https://user-images.githubusercontent.com/90556221/176664279-2a1ac2f1-9a20-4504-bf3f-f600c30aab59.png)

> Yours
![image](https://user-images.githubusercontent.com/90556221/176664438-7a345259-892b-4efc-8291-00f734f87192.png)

Reply:
Resolved